app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,253 @@
+    box-shadow: 0 0 4px 4px rgba(0, 0, 0, 0.03);
+    padding: 1em 0;
+}
+
+.todo_list_title {
+    text-align: center;
+    font-weight: 700;
+    font-size: 1.5rem;
+    text-transform: uppercase;
+    color: white;
+    margin: 0;
+    a {
+        text-decoration: none;
+        color: white;
+        transition: all 0.4s ease-in-out;
+        &:hover {
+            opacity: 0.4;
+        }
+    }
+}
+
+.todo_list_sub_title {
+    margin: 1em 0 3em 0;
+    text-align: center;
+    font-size: 0.9em;
+    color: $dark;
+    font-weight: 100;
+}
+
+.index_row {
+    padding: 2em;
+    border-bottom: 1px solid rgba(250, 250, 250, 0.3);
+    .todo_list_sub_title {
+        margin-bottom: 0;
+    }
+}
+
+#todo_items_wrapper {
+    .row {
+        width: 100%;
+        border-top: 1px solid rgba(250, 250, 250, 0.3);
+        border-bottom: 1px solid rgba(250, 250, 250, 0.3);
+        .complete {
+            width: 10%;
+            float: left;
+            text-align: center;
+            border-right: 1px solid rgba(250, 250, 250, 0.3);
+            padding: 1em 0;
+        }
+        .todo_item {
+            width: 80%;
+            float: left;
+            p {
+                margin: 0;
+                padding: 1em;
+                color: $dark;
+                font-weight: 100;
+            }
+        }
+        .trash {
+            width: 10%;
+            float: left;
+            text-align: center;
+            border-left: 1px solid rgba(250, 250, 250, 0.3);
+            padding: 1em 0;
+        }
+        i {
+            color: white;
+            transition: all 0.4s ease-in-out;
+            &:hover {
+                color: $dark;
+            }
+        }
+    }
+    #form {
+        &::after {
+            content: "";
+            display: table;
+            clear: both;
+        }
+        margin-top: 2em;
+        padding: 0 5%;
+        input[type="text"] {
+            margin-right: 2%;
+            display: inline-block;
+            outline: none;
+            background: rgba(250, 250, 250, 0.4);
+            border: none;
+            height: 40px;
+            border-radius: 4px;
+            padding: 1em 2em;
+        }
+        input[type="submit"] {
+            background: rgba(250, 250, 250, 0.4);
+            outline: none;
+            border: none;
+            height: 40px;
+            border-radius: 4px;
+            width: 25%;
+            display: inline-block;
+            transition: all 0.4s ease-in-out;
+            cursor: pointer;
+            &:hover {
+                background: $dark;
+            }
+        }
+         ::-webkit-input-placeholder {
+            color: $dark;
+        }
+    }
+}
+
+.links {
+    display: block;
+    margin: 1.5em auto 0 auto;
+    text-align: center;
+    font-size: 0.8em;
+    color: white;
+    a {
+        color: white;
+    }
+}
+
+.forms {
+    padding: 0 5%;
+}
+
+label {
+    color: $dark;
+}
+
+input[type="text"],
+textarea {
```

Comment:
Be careful with using HTML tags to apply styles. This styles will apply to the all `textarea`s in the site and can get you in trouble if the project starts growing to the point where specific `textarea` inputs require particular styles. My suggestion is to use CSS classes named after the component + location of the element, for example: `.item-textarea`, `main-textarea`, etc.